.github/workflows/voxelibre.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: buckaroobanzay/mtt@main
         with:
           modname: xcompat
-          git_game_repo: https://git.minetest.land/VoxeLibre/VoxeLibre/
+          git_game_repo: https://github.com/VoxeLibre/VoxeLibre
```

Comment:
@BuckarooBanzay did you fix the host so this could be changed back?

Reply:
should it be changed back? if the github mirror is kept up to date, using github repos for CI could be better since it would reduce how much traffic git.mintest.land has to deal with, also github actions is probably faster accessing github itself.

Reply:
🤷 i mean, how much traffic is just this repo going to send to it?, ill leave it up to buckaroo as to what they want

Reply:
> 🤷 i mean, how much traffic is just this repo going to send to it?, ill leave it up to buckaroo as to what they want

Yes, the access from azure/github-actions should work again (it was a side-effect of some dos-mitigation)

@programmerjake it would be nice if you could revert that back, should work again :+1: